$OpenBSD: patch-Makefile_PL,v 1.1 2018/04/01 23:23:07 bluhm Exp $

Do not check for installed module during make fake,
pkg_add will resolve conflict.

Index: Makefile.PL
--- Makefile.PL.orig
+++ Makefile.PL
@@ -94,59 +94,6 @@ WriteMakefile(
 
 package MY;				## customise generated Makefile
 
-sub install {
-	my $self = shift;
-
-	my %install_type = qw(perl INSTALLPRIVLIB site INSTALLSITELIB vendor INSTALLVENDORLIB);
-	my $install_site = join '', '$(DESTDIR)$(', $install_type{$self->{INSTALLDIRS}}, ')';
-	for ($install_site) {
-		s/\$\(([A-Z_]+)\)/$self->{$1}/eg while /\$\(/;	# expand Makefile macros
-		s|([/])[/]+|$1|g;				# remove gratuitous //s
-	}
-
-	eval 'require Net::DNS::SEC';
-	my @version = grep $_, ( 'version', $Net::DNS::SEC::VERSION );
-
-	my $nameregex = '\W+Net\WDNS\WSEC.pm$';
-	my @installed = grep $_ && m/$nameregex/io, values %INC;
-	my %hidden;
-
-	foreach (@installed) {
-		my $path = $1 if m/^(.+)$nameregex/i;
-		my %seen;
-		foreach (@INC) {
-			$seen{$_}++;				# find $path in @INC
-			last if $_ eq $path;
-		}
-		foreach ( grep !$seen{$_}, @INC ) {
-			$hidden{$_}++;				# hidden libraries
-		}
-	}
-
-	return $self->SUPER::install(@_) unless $hidden{$install_site};
-
-	warn <<"AMEN";
-##
-##	The install location for this version of Net::DNS::SEC differs
-##	from the existing @version in your perl library.
-##	@installed
-##
-##	The installation will be rendered ineffective because the
-##	old version will be found on the library search path before
-##	$install_site
-##
-##	Makefile has been generated to support build and test only.
-##
-AMEN
-
-	return <<'END';
-install :
-	$(NOECHO) $(ECHO) "##	Makefile supports test build only"
-	$(NOECHO) $(ECHO) "##	(see message from Makefile.PL)"
-	$(NOECHO) $(FALSE)
-END
-}
-
 
 sub libscan {
 	my $path = $_[1];
